scripts/needs_ci_run

```diff
@@ -41,17 +43,22 @@ def last_build_successful():
         return False, None
 
     # jobs defined in .circleci/config.yml
-    JOBS = ['lint-build-debug', 'build-release', 'build-osx']
+    JOBS = ['lint-build-debug', 'build-release']
```

Comment:
don't ask me why.. in fact.. should I change the PR to remove the damn thing? so sick of it.. if there's smth not needing any CI for sure, I'd rather quickly cancel those jobs and force merge